src/clients/FlexbaseClient.Person.ts

```diff
@@ -27,12 +28,12 @@ interface PersonResponse extends FlexbaseResponse {
 
 
 export class FlexbaseClientPerson extends FlexbaseClientBase {  
-    async getEmployees(): Promise<Person[]> {
+    async getEmployees(): Promise<PersonUpdateRequest[]> {
         try {
           return await this.client
           .url('/user')
           .get()
-          .json<Person[]>();
+          .json();
 
         } catch (error) {
           console.error('ALL USERS ERROR', error);
```

Comment:
is the pattern to return `null` on error?

Reply:
Yes, it is